src/modules/drawing/components/DrawingTooltip.vue

```diff
@@ -34,113 +44,149 @@ export default {
+                    let pixel = map.getPixelFromCoordinate(coordinate)
+                    return pointWithinTolerance(pixel, event.pixel, hitTolerance)
                 })
-                // detecting if we are hovering the area of a polygon (and not its borders)
-                // if so, we suppress the flag "hoveringSelectedFeature" as it is of no value
-                // to know that we are hovering the area (only borders have interactivity)
-                if (featureUnderCursor?.getGeometry() instanceof Polygon) {
-                    hoveringSelectedFeature = false
+
+                // Display a help tooltip when modifying
+                if (hoveringVertex || ['marker', 'annotation'].includes(featureType)) {
+                    translationKeys = `modify_existing_vertex_${featureType}`
+                } else {
+                    translationKeys = `modify_new_vertex_${featureType}`
                 }
+            } else if (hoveringSelectableFeature) {
+                mapElement.classList.add(cssPointer)
+                mapElement.classList.remove(cssGrab)
 
-                const featureType = featureUnderCursor?.get('drawingMode')?.toLowerCase()
-                if (hoveringSelectedFeature) {
-                    // Display a help tooltip when modifying
-                    if (['marker', 'annotation'].includes(featureType)) {
-                        this.updateTooltipText(`modify_existing_vertex_${featureType}`)
-                    } else {
-                        this.updateTooltipText(`modify_new_vertex_${featureType}`)
-                    }
-                } else if (hoveringSelectableFeature) {
-                    // Display a help tooltip when selecting
-                    if (featureType) {
-                        this.updateTooltipText(`select_feature_${featureType}`)
-                    } else {
-                        this.updateTooltipText('select_no_feature')
-                    }
+                // Display a help tooltip when selecting
+                if (featureType) {
+                    translationKeys = `select_feature_${featureType}`
                 } else {
-                    // Display a help tooltip when drawing
-                    // TODO: help for each drawing mode
-                    if (this.currentDrawingMode) {
-                        this.updateTooltipText(
-                            `draw_start_${this.currentDrawingMode.toLowerCase()}`
+                    translationKeys = 'select_no_feature'
+                }
+            } else {
+                mapElement.classList.remove(cssPointer)
+                mapElement.classList.remove(cssGrab)
+
+                // Display a help tooltip when drawing
+                if (this.currentDrawingMode) {
+                    let type = this.currentDrawingMode.toLowerCase()
+
+                    if (this.currentlySketchedFeature && !['marker', 'annotation'].includes(type)) {
```

Comment:
same as above, I would prefer we use the enum instead of hard coded values

Reply:
Do you mean `EditableFeatureTypes` from features.api.js?

Reply:
I'm not sure if these values ever get serialized or have to be compared to serialized values. But if not we could convert them to symbols.

```js
export const EditableFeatureTypes = {
    MARKER: Symbol('MARKER'),
    ANNOTATION: Symbol('ANNOTATION'),
    LINEPOLYGON: Symbol('LINEPOLYGON'),
    MEASURE: Symbol('MEASURE'),
}
```

Reply:
Hmmm... We'd have to add a mapping for the instances we want to use the value/label as with the translation keys. Probably best to keep it as-is :smiley: